src/third_party/CMakeLists.txt

```diff
@@ -119,4 +119,10 @@ install(
   COMPONENT headers
   FILES_MATCHING PATTERN "*.hpp" PATTERN "*.h"
 )
+add_library(${inDirectory} INTERFACE)
```

Comment:
Can you explain why you need this?

Reply:
At first, I want to declare every third-party utility explicitly so it gives more modularity.
However, just adding 'include/openMVG' to OPENMVG_INCLUDE_DIRS would be enough.